content/news/articles/2025-04-17-opensearch-alerting.md

```diff
@@ -0,0 +1,85 @@
+---
+layout: layouts/post
+tags: news
+title: "Now available: alerting for Cloud.gov logs"
+date: 2025-04-17
+excerpt: How to use the new alerting feature for your Cloud.gov logs
+---
+
+Alerting is now available for the Cloud.gov logging system. This new feature allows users to monitor their logs and send notifications when certain conditions or thresholds are met.
+
+You will be able to setup monitors for different conditions using fields, including:
+
+- The size of a field value
+- The contents of a field value
+- Writing a query to only match under certain conditions(e.g. only in your prod space, only for a specific app)
+
+The potential applications of the new alerting feature include (but are not limited to):
+
+- Monitor for an increase in 404 responses in a certain timeframe
+- Monitor for appearance of certain words (e.g. exceptions, failures) in log contents
+- Monitor when your application's memory usage exceeds a certain threshold
+
+## How access for OpenSearch notifications works
+
+Most of the objects that you create in OpenSearch, such as queries, visualization, or dashboards, are only visible to other members of your OpenSearch tenant. We have configured OpenSearch to have tenants for each cloud.gov organization, meaning there is a one-to-one mapping between tenants and organizations. For whichever tenant you choose when using OpenSearch, only other members of the associated cloud.gov organization can see your objects.
+
+Unfortunately, the OpenSearch alerting plugin [does not support the use of tenants for storing associated objects such as email recipient groups or channels](https://github.com/opensearch-project/alerting-dashboards-plugin/issues/1096).
+
+Given the current behavior of the OpenSearch alerting plugin, to ensure maximum security we have customized OpenSearch so that **only users who share all of the same cloud.gov organizations as your user will be able to see the notification objects that you create**. If another user shares some but not all of the same cloud.gov organizations as you, the names of your notification objects will be visible to the other user, but they will not be able to see any details about those objects.
```

Comment:
```suggestion
Given the current behavior of the OpenSearch alerting plugin, to ensure maximum security we have customized OpenSearch so that **only users who share all of the same cloud.gov organizations as your user will be able to see the notification objects that you create**. If another user shares some but not all of the same Cloud.gov organizations as you, the names of your notification objects will be visible to the other user, but they will not be able to see any details about those objects.
```